galpy/potential/TwoPowerSphericalPotential.py

```diff
@@ -21,8 +24,8 @@ class TwoPowerSphericalPotential(Potential):
 
         \\rho(r) = \\frac{\\mathrm{amp}}{4\\,\\pi\\,a^3}\\,\\frac{1}{(r/a)^\\alpha\\,(1+r/a)^{\\beta-\\alpha}}
     """
-    def __init__(self,amp=1.,a=5.,alpha=1.5,beta=3.5,normalize=False,
-                 ro=None,vo=None):
+    def __init__(self,amp=1.,a=5.,alpha=1.5,beta=3.5,
+                 normalize=False,ro=None,vo=None):
         """
```

Comment:
Please avoid these types of changes (whitespace, linebreaks) in PRs if they have nothing to do with the PR.

Reply:
done.